docs/docs/01-why-react.de-DE.md

```diff
@@ -0,0 +1,28 @@
+---
+id: why-react-de-DE
+title: Warum React?
+permalink: why-react-de-DE.html
+---
+React ist eine JavaScript-Bibliothek von Facebook und Instagram für Benutzeroberflächen. Man kann sich React als das **V** in **[MVC](https://de.wikipedia.org/wiki/Model_View_Controller)** vorstellen.
+
+Wir haben React für folgende Aufgabe entwickelt: **umfangreiche Anwendungen zu bauen, deren Daten sich zeitlich ändern**.
+
+## Einfach
+
+Beschreibe, wie sich Deine App zu jedem gewünschten Zeitpunkt präsentieren soll und React kümmert sich um alle Benutzeroberflächen-Änderungen sobald sich die zugrundliegenden Daten ändern.
```

Comment:
Hi

I wanted to fix the typo myself but honestly, this is my first time ever doing anything on git and I am unsure how to go about this edit.

After that fix, text is ready to be merged. Thanks for reaching out.

Regards
Patrick Thomas 

Von meinem iPhone gesendet

> Am 14.10.2015 um 23:56 schrieb Jim <notifications@github.com>:
> 
> In docs/docs/01-why-react.de-DE.md:
> 
> > @@ -0,0 +1,28 @@
> > +---
> > +id: why-react-de-DE
> > +title: Warum React?
> > +permalink: why-react-de-DE.html
> > +---
> > +React ist eine JavaScript-Bibliothek von Facebook und Instagram für Benutzeroberflächen. Man kann sich React als das **V** in **[MVC](https://de.wikipedia.org/wiki/Model_View_Controller)** vorstellen.
> > +
> > +Wir haben React für folgende Aufgabe entwickelt: **umfangreiche Anwendungen zu bauen, deren Daten sich zeitlich ändern**.
> > +
> > +## Einfach
> > +
> > +Beschreibe, wie sich Deine App zu jedem gewünschten Zeitpunkt präsentieren soll und React kümmert sich um alle Benutzeroberflächen-Änderungen sobald sich die zugrundliegenden Daten ändern.
> @thomasp9 Would like to start closing out PRs like this. Did you want to update this PR to fix the zugrundeliegenden typo and then we can merge, or should we close this PR, or is it fine as-is,, or how do you want to handle this?
> 
> —
> Reply to this email directly or view it on GitHub.
>